packages/ui/src/views/repo/pull-request/components/reviewers/pull-request-reviewers-item.tsx

```diff
@@ -19,9 +19,14 @@ const ReviewerItem = ({ reviewer, reviewDecision, sha, sourceSHA, processReviewD
   }
   return (
     <div key={reviewer?.id} className="flex items-center justify-between space-x-2">
-      <div className="flex items-center space-x-2">
+      <div className="flex items-center space-x-2 overflow-hidden">
         <Avatar name={reviewer?.display_name} rounded />
-        <div className="truncate text-2 font-medium text-cn-foreground-1">{reviewer?.display_name}</div>
+        <Tooltip.Root delayDuration={600}>
```

Comment:
can we replace the Tooltip and the div with the `Text` component instead? it will handle truncation, styling and tooltips automatically.